modules/seqexec/model/src/main/scala/seqexec/model/SequencesBatch.scala

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.model
+
+import cats.Eq
+
+object SequencesBatch {
+  sealed trait CommandState extends Product
```

Comment:
`Product with Serializable`? The point of doing this is to avoid weird LUBs among the cases. Why not also for `State` below?

Reply:
Should this stuff be in the `enum` package since they're enumerated types?

Reply:
To be completely honest, I don't know what `Product` is. I only put it there because otherwise the code would not compile.

Reply:
@tpolecat You mean, to move it to seqexec.model.enum ?